renderer/views/player.js

```diff
@@ -278,18 +304,19 @@ function renderSubtitlesOptions (state) {
     var isSelected = state.playing.subtitles.selectedIndex === ix
     return hx`
       <li onclick=${dispatcher('selectSubtitle', ix)}>
-        <i.icon>${isSelected ? 'radio_button_checked' : 'radio_button_unchecked'}</i>
+        <i.icon>${'radio_button_' + (isSelected ? 'checked' : 'unchecked')}</i>
```

Comment:
this is fine, but not necessarily better because it breaks grepping for `radio_button_checked`